WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1749,6 +1807,14 @@ static GSourceFuncs _handlerIntervention =
         {
             TRACE_L1("%s", url.c_str());
 
+            bool isCurrentUrlBootUrl = urlValue() == getMainConfigValue("app.metroBootPath");
+            bool isNewUrlBootUrl = url == getMainConfigValue("app.metroBootPath");
+            if(!isCurrentUrlBootUrl && isNewUrlBootUrl) {
+                _isAppChangeUrlToBootUrl = true;
+                _bootUrlAppReloadTimePoint = std::chrono::steady_clock::now();
+                ODH_WARNING("WPE0020", WPE_CONTEXT_WITH_URL(url.c_str()), "New URL: %s", url.c_str());
```

Comment:
WPE0020 code we want to report only from URL method, because this is the only place where URL is change is triggered from UI